packages/base-controller/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./",
```

Comment:
Looking through these files you might notice there is a set of properties that all per-package `tsconfig.json` and `tsconfig.build.json` should, and do, have. I tried to move these into the base config (`tsconfig.packages.build.json` or `tsconfig.packages.json`), but as they involve paths, TypeScript seems to interpret those paths relative to the config file where they are specified, and so moving them to a different file results in invalid paths.